src/components/Button.tsx

```diff
@@ -53,8 +55,17 @@ export function Button(props: ButtonProps) {
   const { buttonProps, isPressed } = useButton(
     {
       ...ariaProps,
-      onPress: typeof onPress === "string" ? noop : onPress,
-      elementType: typeof onPress === "string" ? "a" : "button",
+      onPress: asLink
```

Comment:
Just curious, but if the user mashes onPress even while asyncInProgress is true, will this fire again?

Reply:
Oh, I see it; n/m, reviewing on mobile :-)

Reply:
No, once the first `onPress` is invoked, then the button immediately becomes disabled stopping the user from double clicking, or mashing on it.